test/TensorFlow/with_device.swift

```diff
@@ -0,0 +1,32 @@
+// RUN: %target-swift-frontend -Xllvm -tf-use-device-stack -Xllvm -tf-dump-intermediates -Xllvm -tf-dump-graph -Xllvm -tf-module-level-graph=false -emit-sil -verify %s | %FileCheck %s
+
+import TensorFlow
+
+public func deviceFreeFunc() {
+  let y = withDevice(.cpu, 1) { () -> Tensor<Float> in
```

Comment:
```suggestion
  let y = withDevice(.cpu, 1) {
```
Drop redundant type signature?

Reply:
When I remove this, I got a compiler error:

```
swift-base/swift/test/TensorFlow/with_device.swift:6:23: error: unexpected error produced: enum case 'cpu' cannot be used as an instance member
  let y = withDevice(.cpu, 1) {
                      ^
```

Reply:
You can also do:
let y: Tensor<Float> = WithDevice(.cpu, 1) { ... } and it'll solve your problem.